src/overlays/actors/ovl_Dm_Opstage/z_dm_opstage.h

```diff
@@ -3,16 +3,19 @@
 
 #include "global.h"
 
+#define DMOPSTAGE_GET_FF (((this)->dyna.actor.params & 0xFF))
```

Comment:
```suggestion
#define DMOPSTAGE_GET_TYPE(thisx) (((thisx)->params & 0xFF))
```
Gotta include the type in the macro.

Not my call, but the leads want all params macros to try to use thisx instead of this unless they don't match.

Reply:
![image](https://user-images.githubusercontent.com/96613413/172860418-7177bc17-25be-4d41-9e4c-86619fcab695.png)
This is what happens. Also i don't know what is that TYPE thing, the FF info got lost there, i'm just following the tutorial here.

Reply:
None of the code inside there compiles if any DMOPSTAGE_GET_TYPE(thisx) is present...